packages/wallet-connectors/src/WalletConnect.ts

```diff
@@ -71,77 +73,109 @@ function accountTransactionPayloadToJson(data: AccountTransactionPayload) {
     });
 }
 
+function serializeInitContractParam(
+    initName: string,
+    parameters: SmartContractParameters | undefined,
+    schema: Schema | undefined
+) {
+    if (!parameters) {
+        if (schema) {
+            throw new Error(`schema provided when 'parameters' is undefined`);
+        }
+        // No parameters provided.
+        return toBuffer('');
+    }
+    if (!schema) {
+        throw new Error(`schema not provided when 'parameters' is present`);
+    }
+    switch (schema.kind) {
+        case 'module':
+            return serializeInitContractParameters(initName, parameters, schemaAsBuffer(schema.value), schema.version);
+        case 'parameter':
+            return serializeTypeValue(parameters, schemaAsBuffer(schema.value));
+        default:
+            throw new UnreachableCaseError('schema', schema);
+    }
+}
+
+function serializeUpdateContractMessage(
+    contractName: string,
+    receiveName: string,
```

Comment:
`receiveName` is what `foo.bar` is. `contractName` is `foo`, and `entrypointName` is `bar`.

Reply:
The value is passed to [`https://github.com/Concordium/concordium-node-sdk-js/blob/main/packages/common/src/serialization.ts#L488`](https://github.com/Concordium/concordium-node-sdk-js/blob/main/packages/common/src/serialization.ts#L488) where it's called `receiveFunctionName`. It sounds like it should be `entrypointName`?

Reply:
done